project/Http4sPlugin.scala

```diff
@@ -47,6 +49,9 @@ object Http4sPlugin extends AutoPlugin {
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     headerSources / excludeFilter := HiddenFileFilter,
     doctestTestFramework := DoctestTestFramework.Munit,
+    libraryDependencies += scalacCompat,
+    ivyConfigurations += CompileTime,
+    Compile / unmanagedClasspath ++= update.value.select(configurationFilter(CompileTime.name)),
```

Comment:
The magic is borrowed from typelevel/cats#1889.
But perhaps it would be better to put it into `sbt-typelevel`.

Reply:
See also sbt/sbt#2503